andela/wger-bau-meister
# #144208791 Clear exercise cache when deleting muscles

#### What does this PR do?
The cache should be reset after a muscle has been deleted

#### Description of Task to be completed?
After deleting a muscle, it kept appearing in descriptions, muscle overview etc. This is was due to the cache not being correctly reset. After this task, the cache is reset and the deleted muscle does not appear

#### How should this be manually tested?
[https://wger-meister.herokuapp.com](URL)
1. Login to Wger
2. In the```Exercise tab```, go to ```Muscles```. A list of all muscles will be displayed
3. Delete a muscle.
4. In the ```Exercise tab```, go to ```Muscle Overview```. The muscle you deleted will not appear in the list

#### What are the relevant pivotal tracker stories?
144208791
